Wrappers/Python/test/test_quality_measures.py

```diff
@@ -50,9 +51,22 @@ def setUp(self):
 
             self.dc1 = dc1
             self.dc2 = dc2
+            
+            self.mask=ig.allocate(0)
```

Comment:
```suggestion
            self.mask=ig.allocate(0,dtype=bool)
```

Reply:
I have left both types there - a boolean and an integer mask so that I can check both cases